docs/Project-Configuration/Project-Commands.md

```diff
@@ -520,16 +520,18 @@ Agent Parameters:
 
 EC2 Parameters:
 
--   `ami` - EC2 AMI to start. Must set `ami` or `distro` but must not
-    set both.
+-   `ami` - For an `ec2` provider, the AMI to start. Must set `ami` or `distro`
+    but must not set both.
 -   `aws_access_key_id` - AWS access key ID. May set to use a
     non-default account. Must set if `aws_secret_access_key` is set.
 -   `aws_secret_access_key` - AWS secret key. May set to use a
     non-default account. Must set if `aws_access_key_id` is set.
 -   `device_name` - name of EBS device
--   `distro` - Evergreen distro to start. Must set `ami` or `distro` but
-    must not set both. Note that the distro setup script will not run for 
-    hosts spawned by this command, so any required initial setup must be done manually.
+-   `distro` - Evergreen distro to start. For the `ec2` provider, must set
+    either `ami` only) or `distro` but must not set both. For the `docker`
```

Comment:
Nit: need to remove unclosed parentheses here

Reply:
Nice catch, fixed.